adcom1/auto_refresh_trigger.go

```diff
@@ -0,0 +1,12 @@
+package adcom1
+
+// AutoRefreshTrigger represents a list of triggers support by a placement or required by an ad.
+type AutoRefreshTrigger int8
+
+// Triggers support by a placement or required by an ad.
+const (
+	AutoRefreshTriggerUnknown    AutoRefreshTrigger = 0
+	AutoRefreshTriggerUserAction AutoRefreshTrigger = 1 // Refresh triggered by user-initiated action such as scrolling.
+	AutoRefreshTriggerEvent      AutoRefreshTrigger = 2 // Event-driven content change. For example, ads refresh when the football game score changes on the page.
+	AutoRefreshTriggerTime       AutoRefreshTrigger = 3 // Time-based refresh. Ads refresh on a predefined time interval even without user activity.
```

Comment:
Can anyone suggest a more compact prefix? I don't want to just call it `TriggerUnknown`, etc.. as that's too generic and could cause a conflict later.

Reply:
In my opinion, the current form is okay. It may be a little long, but it is informative.

One of the options could be some kind of sub-package, but I think it is not necessary and the result probably wouldn't be satisfactory either.